### ISSUE TITLE
healthcheck should not require deprecated web provider

### ISSUE THREAD

**Issue by wenzowski** [containous/traefik#2524]
Now that the `web` provider has been deprecated, the healthcheck command should probably get its config from `api` instead of depending on web [as documented](https://docs.traefik.io/basics/#command-healthcheck).

It's currently calling [`globalConfiguration.Web.Path`](https://github.com/containous/traefik/blob/master/cmd/traefik/healthcheck.go#L68).

### What did you do?
```
HEALTHCHECK  CMD ["/traefik", "healthcheck", "--web"]
```

### What did you expect to see?
```
OK: http://:8080/ping
```

### What did you see instead?
```
web provider configuration is deprecated, you should use these options : api, rest provider, ping and metrics
OK: http://:8080/ping
```

### Output of `traefik version`: (_What version of Traefik are you using?_)

```
Version:      v1.5.0-rc1
Codename:     cancoillotte
Go version:   go1.9.2
Built:        2017-11-28_02:11:24PM
OS/Arch:      linux/amd64
```

### What is your environment & configuration (arguments, toml, provider, platform, ...)?

```json
{
 "LifeCycle": null,
 "GraceTimeOut": 0,
 "Debug": false,
 "CheckNewVersion": true,
 "SendAnonymousUsage": false,
 "AccessLogsFile": "",
 "AccessLog": null,
 "TraefikLogsFile": "",
 "TraefikLog": null,
 "LogLevel": "ERROR",
 "EntryPoints": {},
 "Cluster": null,
 "Constraints": [],
 "ACME": null,
 "DefaultEntryPoints": [],
 "ProvidersThrottleDuration": 2000000000,
 "MaxIdleConnsPerHost": 200,
 "IdleTimeout": 0,
 "InsecureSkipVerify": false,
 "RootCAs": null,
 "Retry": null,
 "HealthCheck": {
  "Interval": 30000000000
 },
 "RespondingTimeouts": null,
 "ForwardingTimeouts": null,
 "Web": null,
 "Docker": null,
 "File": null,
 "Marathon": null,
 "Consul": null,
 "ConsulCatalog": null,
 "Etcd": null,
 "Zookeeper": null,
 "Boltdb": null,
 "Kubernetes": null,
 "Mesos": null,
 "Eureka": null,
 "ECS": null,
 "Rancher": null,
 "DynamoDB": null,
 "ServiceFabric": null,
 "Rest": null,
 "API": null,
 "Metrics": null,
 "Ping": null,
 "ConfigFile": ""
}
```

Sample [Dockerfile](https://github.com/wenzowski-docker/traefik/blob/master/Dockerfile) using HEALTHCHECK

**Comment by ldez** [comment]
@wenzowski If you use 1.5 please refer to this documentation: http://v1-5.archive.docs.traefik.io/

**Comment by wenzowski** [comment]
Thanks for the link correction @ldez 
Web is still required in docs there too http://v1-5.archive.docs.traefik.io/basics/#command-healthcheck

The point of this issue is that if `--web` or `[web]` in toml is not provided, then the healthcheck fails due to a null reference of `globalConfiguration.Web.Path`. But `[web]` is deprecated.

**Comment by traefiker** [comment]
Closed by #2529.
